content/applications/hr/employees/new_employee.rst

```diff
@@ -358,44 +367,41 @@ regardless.
+     :guilabel:`Create User`. A :guilabel:`Create User` form appears. Type in the :guilabel:`Name`
+     and :guilabel:`Email Address`. Next, select the :guilabel:`Company` from the drop-down menu.
+     Then, enter the :guilabel:`Phone` and :guilabel:`Mobile` numbers in the respective fields. If a
+     photo is available, click the :guilabel:`Edit` icon (which appears as a :guilabel:`✏️ (pencil)`
+     icon) in the lower left corner of the image box in the top right of the form. A file explorer
+     pops up. Navigate to the file, then click :guilabel:`Open` to select it. Finally, click
+     :guilabel:`Save` after all the information is entered, and the employee record is automatically
+     updated with the newly created user populating the :guilabel:`Related User` field.
+
+     Users can also be created manually. For more information on how to manually add a user, refer
+     to the :doc:`../../general/users/manage_users` document.
+
+- :guilabel:`Attendance/Point of Sale/Manufacturing`: the employee's :guilabel:`PIN Code` and
   :guilabel:`Badge ID` can be entered here, if the employee needs/has one. Click
   :guilabel:`Generate` next to the :guilabel:`Badge ID` to create a badge ID. The :guilabel:`PIN
   Code` is used to sign in and out of the *Attendance* app kiosk, and a :abbr:`POS (Point Of Sale)`
   system.
-- :guilabel:`Application Settings`: Enter the employee's :guilabel:`Hourly Cost` in a XX.XX format.
-  This is factored in when the employee is working at a :doc:`work center
-  <../../inventory_and_mrp/manufacturing/management/using_work_centers>`. This value affects the
-  manufacturing costs for a product, if the value of the manufactured product is not a fixed amount.
-  This value does not affect the *Payroll* application. If applicable, enter the :guilabel:`Fleet
-  Mobility Card` number.
+- :guilabel:`Payroll`: if applicable, enter the :guilabel:`Registration Number of the Employee` in
+  this section. Depending on the localization setting, the other items that appear in this field
+  vary based on location. In addition, other sections may appear in this tab based on location. It
+  is recommended to check with the payroll and/or accounting departments to ensure this section, as
+  well as any other sections relating to payroll that may appear, are filled in correctly.
+- :guilabel:`Application Settings`: enter the employee's :guilabel:`Billing Time Target`. This only
+  affects the billing rate leader board in the *Timesheets* application. Next, enter the
+  :guilabel:`Hourly Cost` in a XX.XX format. This is factored in when the employee is working at a
+  :doc:`work center <../../inventory_and_mrp/manufacturing/management/using_work_centers>`. This
+  value affects the manufacturing costs for a product, if the value of the manufactured product is
+  not a fixed amount. This value does not affect the *Payroll* application. If applicable, enter the
+  :guilabel:`Fleet Mobility Card` number.
```

Comment:
I'd probably put the other manufacturing cost details in a note block. And I tried to help with brevity in this paragraph. It's all optional, so feel free to change/ignore it (:

```suggestion
- :guilabel:`Application Settings`: enter the employee's :guilabel:`Billing Time Target`for the
  billing rate leader board in the *Timesheets* application. Next, enter the :guilabel:`Hourly Cost`
  in a decimal format, which affects manufacturing costs at a :doc:`work center
  <../../inventory_and_mrp/manufacturing/management/using_work_centers>`. If applicable, enter the
  :guilabel:`Fleet Mobility Card` number.

  .. note::
     Manufacturing costs are added to the costs for producing a product, if the value of the
     manufactured product is not a fixed amount. This cost does not affect the *Payroll*
     application.
```